series/praha-pyvo/events/2018-06-20.yaml

```diff
@@ -0,0 +1,15 @@
+city: praha
+start: 2018-06-20 19:00:00
+name: Pražské Pyvo
+number: 87
+topic: Python
+description: |
+    Python standardní knihovna
```

Comment:
Nedáme to do topicu?

Reply:
Jo, to mi nějak uniklo

Reply:
Jak u toho není nějaký Makefile, kterým bych snadno rozběhal celý web, abych zkontroloval, že je to ok, tak nemám vždycky energii dohledávat to a dělat to ručně :)

Reply:
Taky z nějakého neznámého důvodu se skutečný title eventu skládá z `name`, `number` a `topic`

Reply:
Vylepšeno a vyšperkováno :)